tui.go

```diff
@@ -41,6 +44,10 @@ func (a App) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	switch msg := msg.(type) {
 	case tea.KeyMsg:
 		if msg.String() == "ctrl+c" || msg.String() == "q" {
+			if a.listView.hasWorkInProgress() && !a.preventedQuitOnce {
```

Comment:
What if there is no work in progress and the user presses quit? Should we not also mark quit pressed once?

Reply:
Then the program exits fully - technically if you started up the application with the same State from main.go, there could be a need but that's not really anything I see a need to account for. Can add, but I don't think its necessary as the tea.Quit should be the last thing that happens within the application.